staff/sys/migrate-vm

```diff
@@ -45,23 +42,25 @@ from ocflib.misc.shell import red
 
 HostAndVm = namedtuple('HostAndVm', ['host', 'vm'])
 
+client = None
+
 
 def exec_command(cmd, ignore_status_code=False):
-    print(bold(cmd))
```

Comment:
The script doesn't print out any of the significant commands it calls as it is, so I deleted this line. If anyone cares they can add a `-v` option.